client/src/utils/API.js

```diff
@@ -9,6 +9,10 @@ export default {
   getUser: function(id) {
     return axios.get("/api/users/" + id);
   },
+//update the user model
+  putUser: function(id) {
```

Comment:
should probably change the name to indicate it is about a card rather than the user info